docs/docs/themes/authoring-themes.md

```diff
@@ -0,0 +1,129 @@
+---
+title: How to author a theme
+---
+
+## What’s contained in this tutorial?
+
+By the end of this tutorial, you’ll have done the following:
+
+- set up your environment for theme development
+- built a Gatsby Theme
+
+## Prerequisites (if any)
+
+Before following this tutorial, you should have experience building a Gatsby site.
+
+> ⚠⚠ Gatsby Themes are currently experimental ⚠⚠
+
+## Setting up your environment for theme development
+
+While there are many ways to develop a Gatsby Theme, we think the easiest way to get started is by creating a Yarn workspace. Yarn workspaces allow you test how a theme might behave inside of a site without you having to actually publish a theme to npm. You are, of course, welcome to develop your theme any way you’d like.
+
+First, if you don’t have yarn installed, you can follow these instructions (https://yarnpkg.com/en/docs/install) to install it.
+
+Next, create a directory for the project:
+
+```bash
+mkdir theme-workspace
+```
+
+`cd` into that folder and create two directories. The first directory will be called the name of the site you use to test your theme. In this tutorial, I’ll call it `sandbox-app`.
+
+```bash
+cd theme-workspace
+mkdir sandbox-app
+```
+
+Then, create a directory called `themes`, and inside it, add a directory called `gatsby-theme-my-theme`.
+
+```bash
+mkdir themes
+cd themes
+mkdir gatsby-theme-my-theme
+```
+
+Conventionally, Gatsby Themes are prefixed with `gatsby-theme-*`. So, if you want to name your theme`awesome`, you can call it`gatsby-theme-awesome`.
+
+Once you have that set up, add a `package.json` file to the root of the project. It should look like this:
+
+```json
```

Comment:
```suggestion
```json:title=package.json
```

Reply:
oh nice! i didn't realize you could do this!